octodns_cloudflare/__init__.py

```diff
@@ -76,6 +77,9 @@ def __init__(self, id, email=None, token=None, cdn=False, retry_count=4,
 
         self._zones = None
         self._zone_records = {}
+        if not self.pagerules:
+            self.SUPPORTS = set(('ALIAS', 'A', 'AAAA', 'CAA', 'CNAME', 'LOC', 'MX', 'NS',
+                    'PTR', 'SRV', 'SPF', 'TXT'))
```

Comment:
I'd prefer not to duplicate the list of supported records twice, once w/o URLFWD, as that opens up the possibility for the two to get out of sync. It's also a little funky that sometimes it's a class static variable and other times a instance property.

I believe it would be preferable to make `SUPPORTS` a dynamically set instance property, something like:


```suggestion
        self.SUPPORTS = set(('ALIAS', 'A', 'AAAA', 'CAA', 'CNAME', 'LOC', 'MX', 'NS',
                             'PTR', 'SRV', 'SPF', 'TXT'))
        if self.pagerules:
            self.SUPPORTS.add('URLFWD')
```

And remove the class static one up above.